Update/patterns featured category product collection

roykho opened the pull request:
<!-- Start by describing the changes made in this Pull Request, and the reason for such changes. -->

<!-- Reference any related issues or PRs here -->

This PR replaces the Unsplash image with a CCO licensed image from Pxhere.

It covers 3 patterns:
* Featured Category / Focused
* Featured Category / Cover image
* Featured Category / Triple

<!-- Don't forget to update the title with something descriptive. -->
<!-- If you can, add the appropriate labels -->

### Screenshots

<!-- If your change has a visual component, add a screenshot here. A "before" screenshot would also be helpful. -->

Featured Category / Focused
| Before | After |
| ------ | ----- |
|![file.png](https://github.com/woocommerce/woocommerce-blocks/assets/2132595/8e29acf0-305c-4de5-aa6f-7d608554dde5)        |![file.png](https://github.com/woocommerce/woocommerce-blocks/assets/2132595/f571a73c-f6c4-4198-b793-da0a29cc73e9)       |

Featured Category / Cover image
| Before | After |
| ------ | ----- |
|![file.png](https://github.com/woocommerce/woocommerce-blocks/assets/2132595/7346b9ff-c6fe-456b-93c0-722a411ac529)        |![file.png](https://github.com/woocommerce/woocommerce-blocks/assets/2132595/650f91bb-400a-441d-a6ea-29fd3d61a204)       |

Featured Category / Triple
| Before | After |
| ------ | ----- |
|![file.png](https://github.com/woocommerce/woocommerce-blocks/assets/2132595/c8b09fd1-0c5c-48eb-9be9-8545f35a37c1)        |![file.png](https://github.com/woocommerce/woocommerce-blocks/assets/2132595/cde1aaf7-0224-40df-93ce-ef1c5640537d)|

### Testing

#### User Facing Testing

<!-- Write these steps from the perspective of a "user" (merchant) familiar with WooCommerce. No need to spell out the steps for common setup scenarios (eg. "Create a product"), but do be specific about the thing being tested. Include screenshots demonstrating expectations where that will be helpful. -->

1. From the editor (post, page, etc.) apply the `Featured Category / Focus, Featured Category / Cover and Featured Category / Triple` pattern.
2. Confirm the image matches the "After" version in the screenshot above in both the admin and the front end.
3. Ensure all CTA links/buttons actually links to the Shop page.


* [ ] Do not include in the Testing Notes <!-- Check this box if this PR can't be tested by users (ie: it doesn't include user-facing changes or it can't be tested without manually modifying the code). -->

### WooCommerce Visibility

<!-- Check this [this doc](../docs/blocks/feature-flags-and-experimental-interfaces.md) to see if the change is visible in WC core or not (part of the feature plugin or experimental)-->

* [x] WooCommerce Core
* [ ] Feature plugin
* [ ] Experimental

### Performance Impact

<!-- Please document any known performance impact (positive or negative) here. If negative, provide some rationale for why this is an okay tradeoff or how this will be addressed. -->

### Changelog

> Update the Featured Category patterns to replace the Unsplash image with a CCO licensed image from Pxhere.

roykho committed:
Update image for featured category focus pattern

roykho committed:
Update image for featured category cover image pattern

roykho committed:
Update images for featured category triple pattern

roykho committed:
Change to wide width

roykho committed:
Update links to buttons

danielwrobert commented on patterns/featured-category-triple.php:
```diff
@@ -6,73 +6,51 @@
-					<strong>Natural body lotions</strong>
+<!-- wp:column {"style":{"spacing":{"padding":{"top":"0px","right":"0px","bottom":"0px","left":"0px"},"blockGap":"0px"}}} -->
+<div class="wp-block-column" style="padding-top:0px;padding-right:0px;padding-bottom:0px;padding-left:0px"><!-- wp:cover {"url":"<?php echo esc_url( plugins_url( 'images/pattern-placeholders/dish-food-baking-dessert-bread-bakery.png', dirname( __FILE__ ) ) ); ?>","id":1,"dimRatio":0,"contentPosition":"bottom center","isDark":false,"style":{"spacing":{"padding":{"bottom":"1.8em","top":"0px","right ...
+<div class="wp-block-cover is-light has-custom-content-position is-position-bottom-center has-base-color has-text-color" style="padding-top:0px;padding-right:0px;padding-bottom:1.8em;padding-left:0px"><span aria-hidden="true" class="wp-block-cover__background has-background-dim-0 has-background-dim"></span><img class="wp-block-cover__image-background wp-image-1" alt="" src="<?php echo esc_url( pl ...
```
Looks like a typo here (an extra `f`) which is causing the text to render black.

danielwrobert replied:
Updated in 83ce4af.

danielwrobert commented on patterns/featured-category-triple.php:
```diff
@@ -6,73 +6,51 @@
+<!-- wp:column {"style":{"spacing":{"padding":{"top":"0px","right":"0px","bottom":"0px","left":"0px"},"blockGap":"0px"}}} -->
+<div class="wp-block-column" style="padding-top:0px;padding-right:0px;padding-bottom:0px;padding-left:0px"><!-- wp:cover {"url":"<?php echo esc_url( plugins_url( 'images/pattern-placeholders/dish-food-baking-dessert-bread-bakery.png', dirname( __FILE__ ) ) ); ?>","id":1,"dimRatio":0,"contentPosition":"bottom center","isDark":false,"style":{"spacing":{"padding":{"bottom":"1.8em","top":"0px","right ...
+<div class="wp-block-cover is-light has-custom-content-position is-position-bottom-center has-base-color has-text-color" style="padding-top:0px;padding-right:0px;padding-bottom:1.8em;padding-left:0px"><span aria-hidden="true" class="wp-block-cover__background has-background-dim-0 has-background-dim"></span><img class="wp-block-cover__image-background wp-image-1" alt="" src="<?php echo esc_url( pl ...
+<p class="has-text-align-center has-text-color has-large-font-size" style="color:#fffffff;margin-top:0;margin-right:0;margin-bottom:0;margin-left:0;padding-top:0;padding-right:0;padding-bottom:0;padding-left:0">
```
Same as above.

danielwrobert replied:
Updated in 83ce4af.

danielwrobert reviewed (commented):
Everything looks good, aside from the hex color typo for the Warm Bread text I noted below.

It's a super small chance so I'll go ahead and push an update for that and merge, since everything else tests as expected.

danielwrobert committed:
Fix typo in hex color value.

There was an extra `f` in the hex color value, causing the text to
unexpectedly render black.

danielwrobert reviewed (approved):
I've updated the hex value typos in 83ce4af. As noted above, everything else LGTM! 🚀
